lib/components/app/responsive-webapp.js

```diff
@@ -163,7 +163,7 @@ class ResponsiveWebapp extends Component {
       navigator.geolocation.watchPosition(
         // On success
         (position) => {
-          receivedPositionResponse({ position })
+          receivedPositionResponse({ position: { ...position } })
```

Comment:
@miles-grant-ibigroup, would you mind adding a comment above this line with the explanation you wrote in this PR, please?